p2p/pubsub/wrapper.go

```diff
@@ -63,7 +70,8 @@ func (ps *PubSub) Register(topic string, handler GossipHandler) {
 }
 
 // Publish message to the topic.
-func (ps *PubSub) Publish(ctx context.Context, topic string, msg []byte) error {
+func (ps *PubSub) Publish(ctx context.Context, protocol string, msg []byte) error {
+	topic := makeTopic(ps.topicPrefix, protocol)
```

Comment:
we capture this topic in metrics, how it will look like in prometheus?

Reply:
it will look like "/xxxxx/1234/bo" for protocol "bo". are you concerned about the data?
to me it's nice if we will be able to see cross-protocol gossip metrics

btw, i will make this prefix value the same as in the noise prologue.